routing_table.go

```diff
@@ -7,6 +7,7 @@ import (
 	"time"
 
 	log "github.com/golang/glog"
+	"github.com/spikebike/nettools"
```

Comment:
On 01/24/2015 04:37 PM, Yves Junqueira wrote:
> In routing_table.go
> <https://github.com/nictuku/dht/pull/32#discussion_r23500747>:
> 
>> @@ -7,6 +7,7 @@ import (
>>  	"time"
>>  
>>  	log "github.com/golang/glog"
>> +	"github.com/spikebike/nettools"
> 
> I've changed this to nictuku/nettools. I hope that doesn't break your
> change.

Ugh, I thought I fixed those, I guess I missed one.

> When doing changes like this, use the original repository in the import
> line - go get the original repo, then add your fork as a remote so you
> can work locally with those changes.

Not sure exactly how to do that actually, but I did intend to revert
that change before issuing the pull.